### ISSUE TITLE
Components: Lists

### ISSUE THREAD

**Issue by rouvenpost** [swisspost/design-system#1580]
- [ ] check list vs. card vs. card list

Component: Cards (Product Cards / Product List) #1579 

**Figma**
List
https://www.figma.com/file/xZ0IW0MJO0vnFicmrHiKaY/Components-Post?type=design&node-id=61%3A371&mode=design&t=dVBMpdDtmgryaufE-1

Cards
https://www.figma.com/file/xZ0IW0MJO0vnFicmrHiKaY/Components-Post?type=design&node-id=2433%3A65729&mode=design&t=v21xVVGydMFRQiHB-1

Card List
https://www.figma.com/file/xZ0IW0MJO0vnFicmrHiKaY/Components-Post?type=design&node-id=14937%3A92525&mode=design&t=v21xVVGydMFRQiHB-1

**Dev**
https://next.design-system.post.ch/?path=/docs/components-card--docs
https://design-system.post.ch/#/bootstrap-samples/card

**Comment by rouvenpost** [comment]
Ready for review

**Comment by alizedebray** [comment]
@rouvenpost 
The list does not seem ready for review yet, should I review the selection list?
If so, is there another ticket for the list?

**Comment by rouvenpost** [comment]
@alizedebray here are the different List combined :https://www.figma.com/design/xZ0IW0MJO0vnFicmrHiKaY/Components-Post?m=auto&node-id=30030-39506&t=XSKgzdxmoPaUNLBT-1

**Comment by oliverschuerch** [comment]
Depends on #2988

**Comment by gfellerph** [comment]
@oliverschuerch is this needed for v1?

**Comment by gfellerph** [comment]
Closing in favour of #3658
